### PR TITLE
fix: format and clean code in modules

### LINKED ISSUE #2435: [Summary] Enhance code style warnings in server
### SubTask

**PS: One PR for multiple modules is fine, but if the total code exceeds **4000** lines, split it into multiple PRs.**

- [x] Adhere to the code conventions outlined in [HugeGraph-Code-Style-Guide](https://github.com/apache/incubator-hugegraph/wiki/HugeGraph-Code-Style-Guide) (please refine the hg-style.xml specification @returnToInnocence) https://github.com/apache/incubator-hugegraph/pull/2457
   - [x] Add the option 'BLANK_LINES_AFTER_CLASS_HEADER' 
   - [x] Add the option 'KEEP_BLANK_LINES'
   - [x] ....
- [x] Correct the copyright format.
	- [x] #2438
	- [x] #2445
- [x] Format the server module 
   - [x] hugegraph-api & cassandra & palo & postgresql & rocksdb & scylladb & test & hbase & mysql (wip #2439)
   - [x] hugegraph-core #2440
   - [x] hugegraph-dist & example #2441
   - [x]  remove extra blank lines https://github.com/apache/incubator-hugegraph/pull/2459
- [x] clean up the code. https://github.com/apache/incubator-hugegraph/pull/2456
- [x] Doc [hugegraph.feishu.cn/wiki/NzaxwWIRHi5mJGkXkR0cHx6mnbg](https://hugegraph.feishu.cn/wiki/NzaxwWIRHi5mJGkXkR0cHx6mnbg)@returnToInnocence
	- [x] Provide doc on how to retain/install multiple versions of IntelliJ IDE 
	- [x] Unified IDE Rules Setup Tutorial 

### Expected & Actual behavior (期望与实际表现)

image
image
image

### TODO
- [ ] IDEA formatting plugin currently lacks support for adding a blank line after enum declarations.image
- [ ] Use [spotless](https://github.com/diffplug/spotless) to format code

### PR DESCRIPTION
**Pull request by msgui**
## Purpose of the PR

- subtask of #2435

## Main Changes

Format & clean code in submodels:

1. API
2. Scylladb
3. Postgresql
4. Rocksdb
5. Palo
6. Mysql
7. Hbase
8. Cassandra
9. Test

## Verifying these changes

- [ ] Trivial rework / code cleanup without any test coverage. (No Need)
- [ ] Already covered by existing tests, such as *(please modify tests here)*.
- [ ] Need tests and can be verified as follows:
  - xxx

## Does this PR potentially affect the following parts?

- [ ] Nope
- [ ] Dependencies (add/update license info) 
- [ ] Modify configurations
- [ ] The public API
- [ ] Other affects (typed here)

## Documentation Status

- [ ] `Doc - TODO` 
- [ ] `Doc - Done` 
- [ ] `Doc - No Need`

### CODE REVIEW

File: hugegraph-server/hugegraph-core/src/main/java/org/apache/hugegraph/backend/cache/CacheManager.java [hunk 1]
```diff
@@ -56,7 +56,7 @@ public static boolean cacheEnableMetrics(String name, boolean enabled) {
         return cache.enableMetrics(enabled);
     }
 
-    public CacheManager() {
+    private CacheManager() {
```
Comment (imbajin) [inline hugegraph-server/hugegraph-core/src/main/java/org/apache/hugegraph/backend/cache/CacheManager.java:59]: emm? why this commit appear again? a little confused

Comment (msgui) [reply hugegraph-server/hugegraph-core/src/main/java/org/apache/hugegraph/backend/cache/CacheManager.java]: thx, it has been corrected.

File: hugegraph-server/hugegraph-cassandra/src/main/java/org/apache/hugegraph/backend/store/cassandra/CassandraMetrics.java [hunk 2]
```diff
@@ -108,7 +108,7 @@ protected Map<String, Object> getMetricsByHost(String host) {
             metrics.put(MEM_UNIT, "MB");
 
             long diskSize = UnitUtil.bytesFromReadableString(
```
Comment (imbajin) [inline hugegraph-server/hugegraph-cassandra/src/main/java/org/apache/hugegraph/backend/store/cassandra/CassandraMetrics.java:110]: seems better keep them in one line (merge into one line if < 101

File: hugegraph-server/hugegraph-cassandra/src/main/java/org/apache/hugegraph/backend/store/cassandra/CassandraShard.java [hunk 3]
```diff
@@ -105,7 +106,7 @@ public List<Shard> getSplits(long splitPartitions, long splitSize) {
                  * compute bite-sized splits.
                  */
                 futures.add(executor.submit(new SplitCallable(
-                            range, splitPartitions, splitSize)));
+                        range, splitPartitions, splitSize)));
```
Comment (imbajin) [inline hugegraph-server/hugegraph-cassandra/src/main/java/org/apache/hugegraph/backend/store/cassandra/CassandraShard.java:109]: same

File: hugegraph-server/hugegraph-cassandra/src/main/java/org/apache/hugegraph/backend/store/cassandra/CassandraStore.java [hunk 4]
```diff
@@ -720,19 +719,19 @@ public CassandraGraphStore(BackendStoreProvider provider,
...
             throw new UnsupportedOperationException(
-                      "CassandraGraphStore.increaseCounter()");
+                    "CassandraGraphStore.increaseCounter()");
         }
 
         @Override
         public long getCounter(HugeType type) {
             throw new UnsupportedOperationException(
-                      "CassandraGraphStore.getCounter()");
+                    "CassandraGraphStore.getCounter()");
```
Comment (imbajin) [inline hugegraph-server/hugegraph-cassandra/src/main/java/org/apache/hugegraph/backend/store/cassandra/CassandraStore.java:734]: same, one line is better but we need do it manually (emmm)

**Review by imbajin — COMMENTED** [review]
A lot of align could be merged into one line (manually), but it will cost some time

File: hugegraph-server/hugegraph-postgresql/src/main/java/org/apache/hugegraph/backend/store/postgresql/PostgresqlTable.java [hunk 5]
```diff
@@ -162,7 +161,7 @@ public PgWhereBuilder(boolean startWithWhere) {
         @Override
         protected String escapeAndWrapString(String value) {
             if (value.equals("\u0000")) {
-                return "\'\'";
+                return "''";
```
Comment (imbajin) [inline hugegraph-server/hugegraph-postgresql/src/main/java/org/apache/hugegraph/backend/store/postgresql/PostgresqlTable.java:164]: better add a test or test it in local env first? (at least maybe add a PR comment here)

File: hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/api/cypher/CypherClient.java [hunk 6]
```diff
@@ -148,10 +148,9 @@ public int hashCode() {
...
     @Override
     public String toString() {
-        final StringBuilder builder = new StringBuilder("CypherClient{");
-        builder.append("userName='").append(userName).append('\'')
-               .append(", token='").append(token).append('\'').append('}');
+        String builder = "CypherClient{" + "userName='" + userName + '\'' +
+                         ", token='" + token + '\'' + '}';
 
-        return builder.toString();
+        return builder;
```
Comment (imbajin) [inline hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/api/cypher/CypherClient.java:154]: ignore the clean for `StringBuilder` (due to perf influence, check if we could config the rule to ignore it)

we could remove the unless SB transfer if we could ensure it's the call frequency is **low**

File: hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/auth/HugeGraphAuthProxy.java [hunk 7]
```diff
@@ -1833,15 +1833,15 @@ protected static Context getContext() {
...
+    private static String getContextString() {
         Context context = getContext();
         if (context == null) {
             return null;
         }
         return context.user().toJson();
     }
 
-    protected static void logUser(User user, String path) {
+    static void logUser(User user, String path) {
```
Comment (imbajin) [inline hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/auth/HugeGraphAuthProxy.java:1844]: consider keeping them first (modification should be separated & careful)

File: hugegraph-server/hugegraph-dist/docker/README.md
Comment (imbajin) [inline hugegraph-server/hugegraph-dist/docker/README.md:1]: ensure no UI diff
image

File: hugegraph-server/hugegraph-test/src/main/java/org/apache/hugegraph/core/VertexCoreTest.java [hunk 8]
```diff
@@ -4115,11 +4116,11 @@ public void testQueryByDoubleProperty() {
...
         final double max15 = new BigDecimal(Double.MAX_VALUE)
                 .movePointLeft(308)
-                .setScale(15, BigDecimal.ROUND_DOWN)
+                .setScale(15, RoundingMode.DOWN)
                 .movePointRight(308)
                 .doubleValue(); // 1.797693134862315E308
-        final double min15 = new BigDecimal(1.234567890987654321d)
-                .setScale(15, BigDecimal.ROUND_DOWN)
+        final double min15 = new BigDecimal("1.234567890987654321")
+                .setScale(15, RoundingMode.DOWN)
```
Comment (imbajin) [inline hugegraph-server/hugegraph-test/src/main/java/org/apache/hugegraph/core/VertexCoreTest.java]: double & string number are the **same case**? need check it carefully

**Review by imbajin — APPROVED** [review]
Most of the files were checked except for **some clean rules** that did not take effect (consider address them together in another PR )

**Review by VGalaxies — APPROVED** [review]

**Pull request merged**